api/falcon/v1alpha1/falconadmission_types.go

```diff
@@ -201,3 +229,35 @@ type FalconAdmissionList struct {
 func init() {
 	SchemeBuilder.Register(&FalconAdmission{}, &FalconAdmissionList{})
 }
+
+func (watcher FalconAdmissionConfigSpec) DeployWatcherContainer() bool {
+	if watcher.DeployWatcher == nil {
+		return false
+	}
+
+	return *watcher.DeployWatcher
+}
+
+func (watcher FalconAdmissionConfigSpec) GetSnapshotsEnabled() string {
+	if watcher.SnapshotsEnabled == nil {
+		return "true"
+	}
+
+	return strconv.FormatBool(*watcher.SnapshotsEnabled)
+}
+
+func (watcher FalconAdmissionConfigSpec) GetSnapshotsInterval() string {
+	if watcher.SnapshotsInterval == nil {
+		return "22h0m0s"
+	}
+
+	return watcher.SnapshotsInterval.Duration.String()
+}
+
+func (watcher FalconAdmissionConfigSpec) GetWatcherEnabled() string {
+	if watcher.WatcherEnabled == nil {
+		return "true"
+	}
+
+	return strconv.FormatBool(*watcher.WatcherEnabled)
+}
```

Comment:
These functions would be more generally useful if they returned their native types (bool, Duration, etc). Places in the code that need them as strings can worry about that themselves.

Reply:
Updated.